### PR TITLE
use L2gateway for address querying in arb-ts

### PR DESCRIPTION
**Pull request by RahulMaganti47**
added functionality to ensure arb-ts looks at l2 router instead of l1

### CODE REVIEW

**Review by DZGoldman — CHANGES_REQUESTED** [review]
Thanks @RahulMaganti47! Run this through the pre-commit hook, and also make sure it builds without errors (`yarn workspace arb-ts build`; it might just be the linting that's triggering the CI error)

File: packages/arb-ts/src/lib/abi/L1ERC20Gateway.d.ts [hunk 1]
```diff
@@ -453,7 +453,7 @@ export class L1ERC20Gateway extends Contract {
 
   counterpartGateway(overrides?: CallOverrides): Promise<string>
 
-  'counterpartGateway()'(overrides?: CallOverrides): Promise<string>
```
Comment (fredlacs) [inline packages/arb-ts/src/lib/abi/L1ERC20Gateway.d.ts:456]: Accidental change?

Comment (RahulMaganti47) [reply packages/arb-ts/src/lib/abi/L1ERC20Gateway.d.ts:456]: Yea my bad

**Comment by DZGoldman** [comment]
merge conflict (also needs a pre-commit pass)

**Pull request closed without merging**